README.md

```diff
@@ -8,10 +8,12 @@ This is CodiTramuntana's Decidim Toolbelt (cdtb), a gem to help managing Decidim
 Install the gem and add to the application's Gemfile by executing:
 
     $ bundle add decidim-cdtb
```

Comment:
This command not works:

```
bundle add decidim-cdtb
Fetching gem metadata from https://rubygems.org/.......
Could not find gem 'decidim-cdtb' in rubygems repository https://rubygems.org/ or installed locally.
```

Reply:
It should work now, I've just published the previous version of the gem. Thanks!